### DISCUSSION TITLE
Prove you're not a robot by solving this quizz....

### DISCUSSION THREAD

**Discussion by FishIsMyNick** [community/community#64402] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Trying to reset my password and this quizz pops up to make me prove I'm human. 
![image](https://github.com/community/community/assets/34579574/45d5487a-44ca-452d-b769-7831136b0b4b)
First it says you have to solve 5 of these. First of all I don't want to take a quick maths quizz with dice to prove I'm human, there are easier ways to do that. Anyways I go for it, because what else can I do at this point. 
It took me longer than I'd like to admit to get all 5 right, and finally when I'm at the end of my misery and think it's all over, github throws a curveball at me. The little bar that fills up as you solve the puzzles extends, telling me I now have to solve 5 more...
Who in their right mind thought this would be even an acceptable idea, let alone a good one...
I realise this might be the wrong place to talk about this, but I couldn't find any link or email to contact support and make my petty complaint. The whole thing was so confusing that I just decided to stop waising time and just post here, at may last attempt to get in touch and say what's on my mind. 
So if you know of a way to contact github support to tell them this myself, please let me know. Thank you.

**Comment by Hyxoo** [comment]
Did u  tried to refresh the quiz by clicking the () button neart the volume button it should generate some new quiz and if u want to contact the GitHub support they won't reply u ig

**Comment by FishIsMyNick** [comment]
Yep, it just reset it all. That’s some QA they got… thanks for the reply
though.
